README.rst

```diff
@@ -169,6 +169,8 @@ NOTE: the output for the following formats are generated in accordance with XML
 
 The ``--verbose`` option will show the applied rules. When using the ``txt`` format it will also display progress notifications.
 
+NOTE: if there is a error like "errors reported during linting after fixing", you can use ```-vvvv``` to be even more verbose for debugging purpose
```

Comment:
In reStructuredText ` ```...``` ` should be ` ``...`` ` (2×2 backquotes, not 2×3).

Reply:
:+1: